publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/controller/ProducerController.java

```diff
@@ -78,12 +83,18 @@ public void setRestTemplate(RestTemplate restTemplate) {
     Logger log = (Logger) LoggerFactory.getLogger(ProducerController.class);
 
     @SuppressWarnings({ "rawtypes", "unchecked" })
+    @ApiOperation(value = "To publish eiffel event to message bus", response = String.class)
+    @ApiResponses(value = { @ApiResponse(code = 200, message = "Event sent successfully"),
+            @ApiResponse(code = 400, message = "Invalid event content"),
+            @ApiResponse(code = 404, message = "RabbitMq properties not found"),
+            @ApiResponse(code = 500, message = "Internal server error"),
+            @ApiResponse(code = 503, message = "Service Unavailable") })
     @RequestMapping(value = "/producer/msg", method = RequestMethod.POST)
     @ResponseBody
-    public ResponseEntity send(@RequestParam(value = "mp", required = false) String msgProtocol,
-            @RequestParam(value = "ud", required = false) String userDomain,
+    public ResponseEntity send(@RequestParam(value = "msgProtocol", required = false) String msgProtocol,
+            @RequestParam(value = "userDomain", required = false) String userDomain,
```

Comment:
its not a good idea to change the reuest parameter names now as they have been alreasy in use from soem time.